.github/workflows/ci.yml

```diff
@@ -38,13 +38,13 @@ jobs:
 
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.12"
+          python-version: "3.8"
```

Comment:
We can add this to `mypy.ini` / `pyproject.toml` instead. So, we won't have to duplicate it and add a detailed doc about it.

Reply:
Are you referring to this? https://github.com/actions/setup-python/pull/669? Given that we specify a range in `requires-python`, will it always select the lowest or is there some way to configure that?

Reply:
No, I was thinking about adding this flag https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-python-version

Reply:
Together / instead of this one